tests/comparisons/Model/testBakeEntityWithPropertyTypeHints.php

```diff
@@ -19,7 +19,7 @@
  * @property \Cake\I18n\FrozenTime|null $updated
  * @property array $array_type
  * @property array $json_type
- * @property $unknown_type
+ * @property string $unknown_type
```

Comment:
Or should this be mixed?

Reply:
Hmm.. I guess `mixed` would be more technically correct. I used `string` as in most cases that would be case and the dev won't have to edit it :slightly_smiling_face:.

I am open to changing it to `mixed` if that's the consensus.

Reply:
At least then people would maybe see it and think about what the actual type might be.
With string it could also be harder to spot the issue.
But either way works.

Reply:
Unless there are custom type mappings being used unknown column types will be fetched as strings.

Reply:
I guess that's true :)

Reply:
So bake needs to support custom types for that to work better?

Reply:
Supporting custom types would be a great improvement by we'd need a hook point for custom types to indicate their property value type.

Reply:
> Unless there are custom type mappings being used unknown column types will be fetched as strings.

Yup, so since PDO returns everything as string that's a reasonable default.

> Supporting custom types would be a great improvement by we'd need a hook point for custom types to indicate their property value type.

That's beyond the scope of this PR :)

Reply:
> That's beyond the scope of this PR :)

Totally agree, I was just thinking out loud.